plugins/ExhibitBuilder/helpers/DPLAFunctions.php

```diff
@@ -644,19 +669,24 @@ function get_id() {
     }
 
     /**
-     * Return the Omeka feild value if it exists
-     * If the Omeka field value does not exists, return the API field value
+     * Return the Omeka or API field value.
+     *
      * @return string|NULL
      * @param string $omekaFieldName 
      * @param array $apiFieldName 
      */
-    private function get_field_value($omeka_field_name, $api_field_name) {
+    private function get_field_value($omeka_field_name, $api_field_name, $api_preferred = false) {
 
         $omeka_field_value = dpla_get_field_value_by_name($this->item, $omeka_field_name);
         $api_field_value = $this->json ? 
             dpla_get_field_value_by_arrayname($this->json, $api_field_name) : null;
+        $field_value = null;
 
-        $field_value = $omeka_field_value ?: $api_field_value;
+        if ($api_preferred == false) {
```

Comment:
The way this is written, it seems like more than one call gets made to get the field value either way. It seems like it'd be a little better to make whatever the first call is, and only fall back to the second if the first is unavailable.